cookbooks/bach_sqoop/templates/default/sq_sqoop-env.sh.erb

```diff
@@ -1,4 +1,6 @@
- Licensed to the Apache Software Foundation (ASF) under one or more
+# -*- mode: shell-script -*-
```

Comment:
Is that bad? Many tools will parse the modeline, and this file has a non-standard filename for a shell script.